gradle-plugin/doc/src/main/asciidoc/inc/generator/_options_common.adoc

```diff
@@ -0,0 +1,44 @@
+There are some configuration options which are shared by all generators:
+
+[[generator-options-common]]
+.Common generator options
+[cols="1,6,1"]
+|===
+| Element | Description | Property
+
+| *add*
+| When set to `true`, then the generator _adds_ to an existing image configuration. By default, this is disabled, so
+  that a generator only kicks in when there are no other image configurations in the build, which are either configured
+  directly for a `{task-prefix}Build` or already added by a generator which has been run previously.
+| `jkube.generator.add`
+
+| *alias*
+| An alias name for referencing this image in various other parts of the configuration. This is also used in the log
+  output. The default alias name is the name of the generator.
+| `jkube.generator.alias`
+
+| *from*
+| This is the base image from where to start when creating the images. By default, the generators make an opinionated
+  decision for the base image which are described in the respective generator section.
+| `jkube.generator.from`
+
+| *fromMode*
+| Whe using OpenShift S2I builds the base image can be either a plain docker image (mode: `jib`) or a reference to an
+  https://docs.openshift.com/container-platform/3.3/architecture/core_concepts/builds_and_image_streams.html[ ImageStreamTag]
+  (mode: `istag`). In the case of an ImageStreamTag, `from` has to be specified in the form `namespace/image-stream:tag`.
+  The mode takes only effect when running in OpenShift mode.
+| `jkube.generator.fromMode`
+
+| *name*
+| The Docker image name used when doing Docker builds. For OpenShift S2I builds its the name of the image stream. This
+  can be a pattern as described in <<image-name, Name Placeholders>>. The default is `%g/%a:%l`. Note that this flag would only work
```

Comment:
In the Maven equivalent, the link is `image-name-placeholders`. I think it might be wrong there? or here? broken link?

Reply:
I'm seeing a broken link in maven docs
![Screenshot from 2021-12-22 21-17-49](https://user-images.githubusercontent.com/13834498/147119046-cf5cedfd-516f-4d93-975d-d5d62d94dd68.png)

Reply:
`image-name` links to:
![image](https://user-images.githubusercontent.com/488391/147119518-97eafd47-8fa5-425c-9d17-2d82c93df36f.png)

It might make sense.

It should be updated in both docs.

Reply:
Shall I update maven plugin doc, then?